test/app.test.js

```diff
@@ -687,76 +688,27 @@ describe('app', function() {
       });
     });
 
-    it('accepts null dataSource', function() {
-      app.model('MyTestModel', { dataSource: null });
-    });
```

Comment:
This test should be preserved. Just pass model ctor instead of the model name, keep the second config arg without changes.